### PR TITLE
complete GitHub OAuth, include front end merge and splitController merger

### PR DESCRIPTION
**Pull request by joekinney-png**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const cookieParser = require('cookie-parser');
 const express = require('express');
 const path = require('path');
-const apiRouter = require('./routes/api');
```
Comment (dsymiller) [inline server/server.js:4]: Bro, we need this. why you delete?

File: server/server.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+const authRouter = require('./routes/auth');
 
 const app = express();
 app.use(express.json());
 app.use(cookieParser());
-// routes all client requests
 
-app.use('/api', apiRouter);
```
Comment (dsymiller) [inline server/server.js:11]: we also need this

File: server/server.js (hunk 1 above)
Comment (joekinney-png) [reply server/server.js:4]: Gp bro

File: server/server.js (hunk 2 above)
Comment (joekinney-png) [reply server/server.js:11]: its still there just spacing

Comment (joekinney-png) [reply server/server.js:11]: oh my b, you right

**Review by dsymiller — APPROVED** [review]
looks good

**Pull request merged**